src/current/cockroachcloud/folders.md

```diff
@@ -75,7 +71,7 @@ The following roles allow creation of clusters at the level of the hierarchy whe
 
 The following additional roles explicitly allow management of folders and their contents:
 
-- **Folder Admins** can create, rename, and move, or delete folders where they are granted the role, and they can also manage access to these folders. This role can be granted at the level of the organization or on a specific folder. If granted on a specific folder, the role is inherited by descendant folders.
+- **Folder Admins** can create, rename, and move, or delete folders where they are granted the role, and they can also manage access to these folders. This role can be granted at the level of the organization or on a specific folder. If granted at the level of the organization, the role grants the ability to view all users and service accounts in the organization. If granted on a specific folder, the role is inherited by descendant folders.
```

Comment:
this information seems duplicative with 'Folder Admins' on the authorization page that is also modified in this PR.  Opportunity  for an include?

suggest mention of 'role' link to something that documents all the cloud roles (if exists)

Reply:
Good idea. This was a little tricky but done.